### ISSUE TITLE
Improve accessibility of tabs

### ISSUE THREAD

**Issue by foster999** [executablebooks/sphinx-tabs#92]
**Describe the bug**

Tabs can't be selected and switched using the keyboard alone.
This would help to comply with [W3C's web accessibility initiative](https://www.w3.org/WAI/WCAG21/quickref/#keyboard).

**To Reproduce**

Use the tab key to move through focusable objects on a page containing sphinx-tabs. Tab sets can't be selected or changed to show content on other tabs.

**Expected behavior**

Users should be able to highlight and switch tabs using the keyboard.
This behaviour is exhibited by [`sphinx-panels` tabbed content](https://sphinx-panels.readthedocs.io/en/latest/#tabbed-content), where the tab key can be used to select the tab set and arrow keys can be used to switch tab.

**Comment by foster999** [comment]
Looking into doing this, but the best approach might be to replace the `a` tag-based tabs here with `button`s and use the [tab role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role).  The example on that link doesn't look too dissimilar to `sphinx-tabs` anyway 😄

A further accessibility issue is that the tab label colour doesn't contrast strongly enough against the background. `#1D5C87` would be the lightest equivalent that meets the standard.

@pradyunsg it might be best to get this sorted before overhauling the JS? Happy to help with either. I'd also be supportive of changing the tabs appearance, but think we should at least provide a theme/option that looks similar to the current tabs for backwards compatibility.

**Comment by pradyunsg** [comment]
> @pradyunsg it might be best to get this sorted before overhauling the JS?

Ahoy! I doubt I'd be coming around to the overhaul anytime soon -- it ended up being easier to write a separate minimal plugin instead. Context [here](https://github.com/executablebooks/meta/issues/145#issuecomment-708667465).

Please do feel free to move ahead without waiting on me to do a rewrite here. :)
